services/movie.py

```diff
@@ -0,0 +1,49 @@
+from typing import Optional
+
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: Optional[list[int]] = None,
+        actors_ids: Optional[list[int]] = None
+) -> QuerySet:
+    queryset = Movie.objects.all()
+
+    if genres_ids and actors_ids:
+        queryset = queryset.filter(
+            genres__id__in=genres_ids
+        ).filter(
+            actors__id__in=actors_ids
+        )
+
+    if actors_ids is None and genres_ids:
+        queryset = queryset.filter(genres__id__in=genres_ids)
+
+    if genres_ids is None and actors_ids:
+        queryset = queryset.filter(actors__id__in=actors_ids)
+
+    return queryset
+
+
+def get_movie_by_id(movie_id: int) -> Movie:

```

Comment:
Consider adding type hinting for the return value of the get_movie_by_id() function to improve code readability. For example: `-> Optional[Movie]:`